test/test_functional_tensor.py

```diff
@@ -418,21 +418,21 @@ def test_affine(self):
         test_configs = [
             (45, [5, 6], 1.0, [0.0, 0.0]),
             (33, (5, -4), 1.0, [0.0, 0.0]),
-            (45, [5, 4], 1.2, [0.0, 0.0]),
-            (33, (4, 8), 2.0, [0.0, 0.0]),
+            (45, [-5, 4], 1.2, [0.0, 0.0]),
```

Comment:
Is there a particular reason why we are changing those values?

Reply:
Tests were missing negative transpose values, so added just them.